Game/Source/GDKTestGyms/Private/UserExperienceReporter.cpp

```diff
@@ -96,6 +98,10 @@ void UUserExperienceReporter::ReportMetrics()
 			if (Constants->ClientFPSMetricDelay.HasTimerGoneOff() && GameInstance->GetAveragedFPS() < Constants->GetMinClientFPS())
 			{
 				bFrameRateValid = false;
+				if (const USpatialNetDriver* NetDriver = Cast<USpatialNetDriver>(GameInstance->GetWorld()->GetNetDriver()))
+				{
+					UE_LOG(LogTemp, Error, TEXT("Client %s failed with fps %f below min threshold %f"), *NetDriver->Connection->GetWorkerId(), GameInstance->GetAveragedFPS(), Constants->GetMinClientFPS());
```

Comment:
`LogTemp` ?

Reply:
Yeah - we use it here and elsewhere in small classes within the test gyms